lib/tasks/import_staticstats.rake

```diff
@@ -0,0 +1,93 @@
+require 'csv'
+
+namespace :import do
+  desc "import CSV data into database"
+  task :staticstats, [:csv_file] => [:environment] do |t, args|
```

Comment:
seems you are not using `t` or `args` anywhere in the code. I think you can safely omit them

Reply:
Good point! I guess it would be useful for passing arguments but we aren't in this case!

Reply:
I tried deleting those but I can't seem to get the rake task to run now!